templates/corporate/comparison_table_integrated.html

```diff
@@ -488,7 +491,7 @@ <h2>Self-hosted plans comparison</h2>
             </tr>
             <tr>
                 <td class="comparison-table-feature">
-                    <a href="/help/bots-and-integrations">
+                    <a href="/help/bots-overview">
```

Comment:
This hunk probably belongs in the previous commit.

Reply:
Oops, yeah. I resolved that change from the old files in the wrong spot. I'll fix that.